front/app/components/admin/PostManager/InputManager.tsx

```diff
@@ -225,7 +225,13 @@ const InputManager = ({
   };
 
   const onResetParams = () => {
-    setQueryParameters({});
+    setQueryParameters(
```

Comment:
Even better: move queryParameters management one level up to distinguish between general and project input manager more easily. Then also pass down queryParameters and onChangeQueryParameters instead of the quadrillion props we have now. But that would take a few hours probably. :)